test/fixtures/disable-warnings-worker.js

```diff
@@ -0,0 +1,3 @@
+const path = require('node:path');
```

Comment:
```suggestion
'use strict';
const path = require('node:path');
```

Reply:
We don't usually add this for fixtures?

Reply:
Any reason this suggestion was not applied? To be clear it's just a nit, so feel free to ignore, but I'm curious.

Reply:
was wondering if there was anything more to Geoffrey's question. I can add it now since I'm changing the option name